### ISSUE TITLE
need python 3.12 release

### ISSUE THREAD

**Issue by fbergmann** [sys-bio/libOmexMeta#148]
@luciansmith  Currently there is no release of pyomexmeta for python 3.12, it would be great if this could be added. Especially since it is a requirement for the biosimulators to run.

**Comment by luciansmith** [comment]
I will work on this!  Looks like I'll either need to update the docker image with Py312 and build on azure, or move to github actions like we've done with everything else.

**Comment by nickerso** [comment]
@luciansmith - did this get done?

**Comment by luciansmith** [comment]
Nope!  But I believe I dropped the libomexmeta requirement for biosimulators.  Thanks for the ping; it's back on my radar.

**Comment by matthiaskoenig** [comment]
Perhaps if this addresses also the py3.13 release could be added.

**Comment by luciansmith** [comment]
So, I fumbled around trying to make a new CI for this, when I realized that this doesn't use Swig:  it uses CTypes.  And CTypes is python-version agnostic.  So!  I tried renaming the wheel files, which worked locally, but couldn't be uploaded to pypi.  So I edited the wheel files directly to get a new version number, and upload those, which... seems to have worked?  It works locally for me, but if others could check to see if it works for them, that'd be great.  Thank you!

**Comment by nickerso** [comment]
I've updated https://github.com/sys-bio/pyomexmeta-binder-notebook to use Python 3.12 and the latest pyomexmeta and that seems to be working well on binder.
